_targets/build.project.armv7m7-imxrt117x

```diff
@@ -54,13 +54,17 @@ KERNEL_PHBASE=$((0x0)) # itcm start address
 KERNEL_PHOFFS=$((0x0))
 export KERNEL_PHADDR=$(printf "%08x" $((KERNEL_PHBASE + KERNEL_PHOFFS)))
 
+# All data/code maps need to be cacheable to allow unaligned accesses
 
 # Pre-init script is launched before user script
 PREINIT_SCRIPT=(
-	"map itcm 0x0 0x40000 rwx"
-	"map dtcm 0x20000000 0x20028000 rwx"
-	"map ocram2 0x202c0000 0x20340000 rwxc"
-	"map xip1 0x30000000 0x30400000 rx"
+	"map itcm 0x0 0x40000 rwxcb"
+	"map dtcm 0x20000000 0x20028000 rwcb"
+	"map ocram2 0x202c0000 0x20340000 rwxcb"
+	"map aips14 0x40000000 0x41000000 rws"
+	"map aips5 0x42000000 0x42100000 rws"
+	"map xip1 0x30000000 0x30400000 rxcb"
+	"map flexspi 0x7f000000 0x80000000 rws"
```

Comment:
I think we don't use this range directly, also `imxrt-flash` accesses flexspi through `0x400CC000 ..+0x460` (FSPI1), `0x400D0000 ..+0x460` (FSPI2) and this is `AIPS-4` range (imxrt117x case)

Reply:
Removed